README.md

```diff
@@ -32,6 +32,7 @@ To try OpenSpiel in Google Colaboratory, please refer to `open_spiel/colabs` sub
 Please choose among the following options:
 
 *   [Installing OpenSpiel](docs/install.md)
+*   **[Windows Installation (pip install)](WINDOWS_INSTALL.md)**
```

Comment:
Also please remove it from the main page as the WIndows install instructions are linked from docs/install.md already.